packages/plugin-react-native-unhandled-rejection/rejection-handler.js

```diff
@@ -16,8 +18,20 @@ module.exports = {
           severityReason: { type: 'unhandledPromiseRejection' }
         }, 'promise rejection tracking', 1)
         client._notify(event)
+        // adding our own onUnhandled callback means the default log message doesn't happen, so make it happen here
+        if (typeof __DEV__ !== 'undefined' && __DEV__) logError(id, error)
       }
     })
     return () => rnPromise.disable()
   }
 }
+
+// this function is copied in from the promise module, since it's not exported and we can't reference it:
```

Comment:
Not sure if we need to add the license here too; I guess this isn't a "substantial portion" so maybe not?

Reply:
It doesn't seem that substantial to me but IANAL 🤷